Purchases/SubscriberAttributes/RCSubscriberAttributesManager.m

```diff
@@ -115,15 +115,15 @@ - (void)markAttributesAsSynced:(RCSubscriberAttributeDict)syncedAttributes
     }
 }
 
-- (void)storeAttributeLocallyIfNeededWithKey:(NSString *)key value:(NSString *)value appUserID:(NSString *)appUserID {
+- (void)storeAttributeLocallyIfNeededWithKey:(NSString *)key value:(nullable NSString *)value appUserID:(NSString *)appUserID {
     NSString *valueOrEmpty = value ?: @"";
     NSString * _Nullable currentValue = [self currentValueForAttributeWithKey:key appUserID:appUserID];
     if (!currentValue || ![currentValue isEqualToString:valueOrEmpty]) {
         [self storeAttributeLocallyWithKey:key value:value appUserID:appUserID];
```

Comment:
This looks like a bug - we should be calling the `storeAttributeLocallyWithKey` with `valueOrEmpty` instead of `value`. 

Could you update it? 

```
[self storeAttributeLocallyWithKey:key value:valueOrEmpty appUserID:appUserID];
```

Note that calling it this way also removes the need for `storeAttributeLocallyWithKey:value:appUserID` to take a nullable value, and in turn the need for `RCSubscriberAttribute` to accept nil values.

Reply:
This indeed looks like a bug. I applied the change to use valueOrEmpty and removed the need to support nullable parameters.